plugins/woocommerce/src/Blocks/BlockTypes/ProductFiltersOverlayNavigation.php

```diff
@@ -11,4 +13,80 @@ class ProductFiltersOverlayNavigation extends AbstractBlock {
 	 * @var string
 	 */
 	protected $block_name = 'product-filters-overlay-navigation';
+
+	/**
+	 * Get the frontend script handle for this block type.
+	 *
+	 * @see $this->register_block_type()
+	 * @param string $key Data to get, or default to everything.
+	 * @return array|string|null
+	 */
+	protected function get_block_type_script( $key = null ) {
+		return null;
+	}
+
+	/**
+	 * Include and render the block.
+	 *
+	 * @param array    $attributes Block attributes. Default empty array.
+	 * @param string   $content    Block content. Default empty string.
+	 * @param WP_Block $block      Block instance.
+	 * @return string Rendered block type output.
+	 */
+	protected function render( $attributes, $content, $block ) {
+		$classes_and_styles = StyleAttributesUtils::get_classes_and_styles_by_attributes( $attributes );
+		$classes            = $classes_and_styles['classes'] . ' wc-block-product-filters-overlay-navigation';
+		$wrapper_attributes = get_block_wrapper_attributes(
+			array(
+				'class' => trim( $classes ),
+				'style' => trim( $classes_and_styles['style'] ),
```

Comment:
This throws a warning:

`Warning: Undefined array key "style" in /Users/dev/workspace/woocommerce/plugins/woocommerce/src/Blocks/BlockTypes/ProductFiltersOverlayNavigation.php on line 42`

Reply:
Fixed in 13eda9598a568ef3aa134dcd2ff311442b4e934c